securedrop_client/api_jobs/uploads.py

```diff
@@ -32,7 +32,6 @@ def call_api(self, api_client: API, session: Session) -> str:
         try:
```

Comment:
note: all exception handling is now in this try block, which i find easier to read